spec/services/exports/payouts/csv_spec.rb

```diff
@@ -108,8 +110,8 @@
         ["Partial Refund", (@purchase_to_refund_partially.succeeded_at + 1.day).to_date.to_s, @purchase_to_refund_partially.external_id, @purchase_to_refund_partially.link.name, @purchase_to_refund_partially.full_name, @purchase_to_refund_partially.purchaser_email_or_email, "-0.0", "-0.0", "-3.5", "-0.55", "-2.95"],
         ["Full Refund", (@purchase_to_refund_from_years_ago.succeeded_at + 1.day).to_date.to_s, @purchase_to_refund_from_years_ago.external_id, @purchase_to_refund_from_years_ago.link.name, @purchase_to_refund_from_years_ago.full_name, @purchase_to_refund_from_years_ago.purchaser_email_or_email, "-0.0", "-0.0", "-10.0", "-1.5", "-8.5"],
         ["PayPal Payouts", @payout.payout_period_end_date.to_s, "", "", "", "", "", "", "-6.5", "", "-6.5"],
-        ["Payout Fee", @payout.payout_period_end_date.to_s, "", "", "", "", "", "", "", "0.76", "-0.76"],
-        ["Totals", nil, nil, nil, nil, nil, "2.0", "0.0", "46.14", "9.16", "36.98"]
+        ["Payout Fee", @payout.payout_period_end_date.to_s, "", "", "", "", "", "", "", "0.92", "-0.92"],
+        ["Totals", nil, nil, nil, nil, nil, "3.8", "0.0", "56.14", "11.41", "44.73"]
```

Comment:
- Total taxes changed from $2.0 to $3.8 because of the $1.8 tax on the newly added `@purchase_with_gumroad_tax`.
- Total price changed from $46.14 to $56.14 because `@purchase_with_gumroad_tax`'s price is $10.
- Total fees changed from $9.16 to $11.41 because $2.09 is the fees on `@purchase_with_gumroad_tax` and $0.16 added 2% payout fee ($0.92 instead of $0.76).